packages/core/phase1/triggers/TRPR0003.test.ts

```diff
@@ -0,0 +1,209 @@
+import generator from "./TRPR0003"
+import generateAhoFromOffenceList from "../../phase2/tests/fixtures/helpers/generateAhoFromOffenceList"
+import type { Offence } from "../../types/AnnotatedHearingOutcome"
+
+describe("TRPR0003", () => {
+  it("should return a trigger if an offence has result codes that exist in the mainResultCodes list", () => {
```

Comment:
Something weird going on with the formatting in this file - looks like there's an extra comma after each `it` block, though I'm not sure how it's valid code